Gifski/MainWindowController.swift

```diff
@@ -307,6 +308,12 @@ final class MainWindowController: NSWindowController {
 			timeRemainingLabel.topAnchor.constraint(equalTo: circularProgress.bottomAnchor)
 		])
 	}
+
+	private func addGifskiMetadata(to url: URL) {
+		let app = "\"\(App.name) \(App.versionWithBuild)\""
+		let itemCreatorKey = kMDItemCreator as String
+		setxattr(url.path, "com.apple.metadata:\(itemCreatorKey)", app, strlen(app), 0, 0)
```

Comment:
I'm surprised this works. The first argument of `setxattr` is typed as `UnsafePointer<Int8>!`. Not sure if it's safe though.

You also need to handle POSIX errors.